velox/functions/sparksql/CharTypeWriteSideCheck.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <string>
+
+#include "velox/functions/Macros.h"
+#include "velox/functions/lib/string/StringImpl.h"
+#include "velox/functions/sparksql/CharVarcharUtils.h"
+
+namespace facebook::velox::functions::sparksql {
+
+/// Ensures that `'abc'` fits within the specified length `n` in
```

Comment:
nit: there's no definition for 'abc' and 'n', perhaps just use `input string` and `limit` to replace.